docker/docker-compose.yaml

```diff
@@ -10,7 +13,10 @@ services:
       - TEXTILE_LOTUS_TOKEN=${TEXTILE_LOTUS_TOKEN}
     ports:
       - 8888:8888
+      - 6060:6060
     restart: always
+    volumes:
+      - textile-fc-data:/root/.texfc
```

Comment:
Made a change in the docker-compose used to launch the server, prometheus, grafana, etc.
The `textile-fc` (our service) uses a volume to map the repo folder to a Docker volume. Mostly for saving progress while restarting the compose environment, and tests index persistence.